packages/flutter/lib/src/widgets/icon.dart

```diff
@@ -69,15 +55,16 @@ class Icon extends StatelessWidget {
 
   /// The color to use when drawing the icon.
   ///
-  /// Defaults to the current [IconTheme] color, if any. If there is
-  /// no [IconTheme], then it defaults to white if the theme is dark
-  /// and black if the theme is light. See [Theme] to set the current
-  /// theme and [ThemeData.brightness] for setting the current theme's
-  /// brightness.
+  /// Defaults to the current [IconTheme] color, if any.
   ///
   /// The given color will be adjusted by the opacity of the current
   /// [IconTheme], if any.
   ///
+  /// In material apps, if there is no [IconTheme], then it defaults to
+  /// white if the theme is dark and black if the theme is light.
+  /// See [Theme] to set the current theme and [ThemeData.brightness]
+  /// for setting the current theme's brightness.
```

Comment:
Rephrased a bit. Left some references to Theme since inserting a non icon theme specifying Theme still changes the color